### ISSUE TITLE
Clean HEP models

### ISSUE THREAD

**Issue by KaziPlays** [BEEmod/BEE2-items#4082]
### Description of the feature

1- New HEP Models:

Discord user asd417#9974 created these Portal 2 styled HEP models, and I think they would be a good fit for the Clean style.

![hep_set](https://user-images.githubusercontent.com/72144613/153726543-15389de4-8764-46b9-9493-e3a1494c02d9.PNG)

2- New HEP Sounds:

I took inspiration from Lego Dimensions for this one. In Lego Dimensions, the HEP sounds are not the same classic sounds from P1, but different sounds.

Use this video as example, 17:54.

https://youtu.be/wo-9lBiUlCc

### Why should this be implemented

HEPs are definitely underrated and underused in the franchise, so I wanted to give it some little improvements to pay respects to the classics. Or something like that...

**Comment by Luke18033** [comment]
I can't find any ripped copies of the Lego Dimensions sounds, although most of them just seem to be electrical spark sound effects anyway so we probably won't use them...

I would like to implement those model, although IIRC there are some slight issues with them (missing/badly textured faces, and the catcher is missing the attachment point for the caught ball)

**Comment by KaziPlays** [comment]
IIRC Alive and kicking used spark sounds for the pellet as well

**Comment by asd417** [comment]
What can I do to make this happen? The model has blue orange and green skin

**Comment by Luke18033** [comment]
Ah, the green skins are used for infinite launchers and timed catchers (like in Rexaura). We'll want to add a third skin to the models with the lights recolored to green.

**Comment by asd417** [comment]
Yeah the model already has green skin. Maybe I should make the overgrown skin?

**Comment by Luke18033** [comment]
Sorry for the late response but yeah they do need overgrown skins. (We could have it use the P1 design still since it's supposed to be destroyed P1 chambers, but that would be inconsistent with every other test element...)

**Comment by KaziPlays** [comment]
The clean pellets could now use the portal rtx model

**Comment by Luke18033** [comment]
Those are completely different, they're still based on the P1 design.

**Comment by KaziPlays** [comment]
Another problem fixed then, there will be no inconsistency with the overgrown model.

**Comment by Luke18033** [comment]
The point of using asd's models is to match the designs of the P2 test elements, not to just be a generic "high quality" version. They/we would want to just add overgrown skins to them.

**Comment by KaziPlays** [comment]
The asset has been updated several times, I think it's time this gets added if it hasn't been yet. If it has, close this.

**Comment by LincoCompany00** [comment]
The Portal: Alive and Kicking mod has some really good Portal 2-style elements, especially high energy pellet models and Unstationary Scaffolds.

**Comment by LincoCompany00** [comment]
The blue lights on the catcher should turn orange when it's activated, like a weighted button. For the emmitter, the lights would be blue if it has no inputs. But if it has an input to turn it on and off, it should turn orange when active and blue when deactive. Or that's how I imagine it, at least.

**Comment by LincoCompany00** [comment]
If the Portal: Alive and Kicking models are replicated and used, than it will match pretty close to the normal overgrown texture, and it will be portal 2 styled.

**Comment by LincoCompany00** [comment]
For the the overgrown models, maybe it could have an option to match clean or portal 1? I don't think the  sounds should be changed much though, as with Lego Dimensions or Alive and Kicking, they're just plain old sparks noises, while P1 they actually have their own unique sound. I'm sure you already discussed this though

**Comment by LincoCompany00** [comment]
The model would be more Portal 2 if this part here (red) was this gray (green) as well (unless it's on it already in the picture where I can't see it)
![Image](https://github.com/user-attachments/assets/4ad82f00-c591-4512-8fe6-baa8bc8d0fe1)
I'm just stating my ideas and opinions, by the way. I'm not trying to take over the discussion or anything, I'm just stating my thoughts to see if they help

**Comment by LincoCompany00** [comment]
Never mind, I found asd417#9974's github repository. Thanks!
I mean, it's okay if I use them, right?

Yeah, now that I look closer at the Alive and Kicking HEP models, asd417's are way better.

**Comment by LincoCompany00** [comment]
You're right, now that I've played Alive and Kicking firsthand I can see that their high energy pellet models really aren't that great, my mistake. However I think the sounds used for Unstationary Scaffolds are a great fit for BEE2's clean style!
